### ISSUE TITLE
[ACCESSIBILITY] Radio buttons in search aren't labelled as such and are non-discoverable by screenreaders

### ISSUE THREAD

**Issue by dommitchell** [DOAJ/doaj#2416]
**Your contact details** 
@spudthebud - Mark logged this with us via Help Desk.

**Where this happened**
Journal homepage: journals/articles search. Mark tested in both JAWS and NVDA. Both screenreaders were unable to identify the radio buttons. They said that radio buttons didn't exist.

**What happened**
See attached video 

https://github.com/user-attachments/assets/b72f64c9-013f-47ff-b539-e626299f662f

**Expected behaviour**
Radio buttons on the homepage should be identifiable as such by the screenreaders. The expected behaviour is that the screen reader should say something like 'Journals radio button: toggled'.

**How to reproduce**
See video

**Desktop (please complete the following information):**
Chrome

**Comment by dommitchell** [comment]
Hi @amdomanska, I'm moving this to the PM repo, and we can work on it there. Thank you!
